docs/installation/gateways.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Swarm Gateways
+id: gateway
+---
+
+For now, not all of your devices, friends and users of your app will be able to run a Bee node which is able to connect directly to the network using *libp2p*. 
+
+To access the Swarm network in these situations, it may be useful for you to provide a Swarm Gateway, which enables devices with only access to traditional web2 TCP/HTTP connections access to some elements of the swarm.
+
+## Public Gateways
+
+The Swarm Foundation provide access to various public gateways.
+
+### Swarm Foundation Gateway
+
+Use the [Swarm Gateway](https://gateway.ethswarm.org/) to try out Swarm, you are able to upload up to 10mb at a time for free. Upload files, directories, or even folders containing websites with and `index.html` file to host them on Swarm.
+
+You will receive an access link you can then share with others who will be able to retrieve the uploaded file, or browse your website directly from any gateway.
+
+`https://gateway.ethswarm.org/`
+
+### Swarm Foundation Gateway API
+
+If you would like to try out *swarm-cli* or *bee-js* you can start by specfiying the public gateway for operations that do not require access to the *Debug API*.
```

Comment:
```suggestion
You can also use the public gateways to access Swarm through *swarm-cli* or *bee-js*. This may be sufficient for some use cases, but many will require access to *Debug API*, which is not exposed on any gateway.
```